sympy/physics/continuum_mechanics/beam.py

```diff
@@ -29,7 +29,10 @@ class Beam:
     .. note::
        While solving a beam bending problem, a user should choose its
        own sign convention and should stick to it. The results will
-       automatically follow the chosen sign convention.
+       automatically follow the chosen sign convention. However, the
+       chosen sign convention must respect the rule that on the positive
+       axis side of the section, a force giving positive shear yields a
+       negative moment.
```

Comment:
I think this note might not be sufficient enough and beginner friendly in explaining what sign convention the user has to choose. Maybe adding an example might help. Something like this:
```
 For Example:   
    Given a right handed coordinate system with XYZ coordinates,
    the beam's length is assumed to be along the positive X axis.
    The loading forces are directed either in the positive or
    negative Y direction and the loading moments are about
    the positive or negative Z direction.
    The only thing that should be considered while choosing
    sign convention is that forces acting along positive
    Y direction(upward in this case) and moments acting along
    negative z direction(clockwise in this case) must have the same signs.
```
What are your thoughts?

Reply:
I have seen a lot of new users make this kind of mistake and getting wrong results with the code.

Reply:
Describing in a few words the allowed sign conventions it's not easy, maybe it's better to add an image like the following one. The curved arrow show the positive rotation. In orange is shown a possible (among many) alternate name for axes.

![github-allowed-signes-convention-opt](https://user-images.githubusercontent.com/4848618/102800319-fe532180-43b3-11eb-87d3-5de705a7d475.png)

Reply:
Yes that would be better. Go ahead!

Reply:
I've updated the note about allowed sign conventions and added a description image, but I don't know how to check the result through Sphinx (I guess).

Reply:
[See this.](https://docs.sympy.org/latest/documentation-style-guide.html?highlight=make%20html#build-the-documentation)
Please build the documentation and open the html file in a browser to see if the image is inserted or not. 
Also add a screenshot here of the webpage showing the image with the note.

Reply:
Running `make html` I get an error:

```
rm -rf _build/logo
mkdir -p _build/logo
python ./generate_logos.py -d
INFO:root: File saved: C:\DATI\ingegneria\code\sympy-nopria\sympy-patch-1\doc\_build/logo\sympy-notail.svg
INFO:root: File saved: C:\DATI\ingegneria\code\sympy-nopria\sympy-patch-1\doc\_build/logo\sympy-notailtext.svg
INFO:root: File saved: C:\DATI\ingegneria\code\sympy-nopria\sympy-patch-1\doc\_build/logo\sympy-notext.svg
ERROR:root:Return code is not 0: Command: rsvg-convert C:\DATI\ingegneria\code\sympy-nopria\sympy-patch-1\doc\src/logo\sympy.svg -f png -o C:\DATI\ingegneria\code\sympy-nopria\sympy-patch-1\doc\_build/logo\sympy-160px.png -h 160 -w 160
ERROR:root:return code: 1
mingw32-make: *** [Makefile:144: logo] Error 1
```

Reply:
There must be some unfulfilled dependencies. Let me build it on my system.

Reply:
I tried building it. The image is present in the wrong directory. Please move it to `sympy/doc/src/modules/physics/continuum_mechanics/` and delete the one in the current directory.

Reply:
I applied your commit suggestion and moved the image file to `sympy/doc/src/modules/physics/continuum_mechanics/`, but running `make html` I still get an error:

```
(nopria-sympy) C:\DATI\ingegneria\code\sympy-nopria\sympy-patch-1\doc>make html
rm -rf _build/logo
mkdir -p _build/logo
python ./generate_logos.py -d
INFO:root: File saved: C:\DATI\ingegneria\code\sympy-nopria\sympy-patch-1\doc\_build/logo\sympy-notail.svg
INFO:root: File saved: C:\DATI\ingegneria\code\sympy-nopria\sympy-patch-1\doc\_build/logo\sympy-notailtext.svg
INFO:root: File saved: C:\DATI\ingegneria\code\sympy-nopria\sympy-patch-1\doc\_build/logo\sympy-notext.svg
ERROR:root:Return code is not 0: Command: rsvg-convert C:\DATI\ingegneria\code\sympy-nopria\sympy-patch-1\doc\src/logo\sympy.svg -f png -o C:\DATI\ingegneria\code\sympy-nopria\sympy-patch-1\doc\_build/logo\sympy-160px.png -h 160 -w 160
ERROR:root:return code: 1
mingw32-make: *** [Makefile:144: logo] Error 1
```

Reply:
![Screenshot from 2020-12-24 00-29-20](https://user-images.githubusercontent.com/50932406/103029425-cc8eb600-457f-11eb-91bc-17a693b96dd3.png)
Its working now.

Reply:
> I applied your [commit suggestion](https://github.com/sympy/sympy/pull/20431#discussion_r548012444) and moved the image file to `sympy/doc/src/modules/physics/continuum_mechanics/`, but running `make html` I still get an error:

I think you havent installed the dependencies required for sphinx properly. Have you installed librsvg?

Reply:
No, I didn't. After installing librsvg I still get other errors, but I don't care as long as it worked for you. Glad to know `beam.py` will finally give correct results. Next I'd like to improve `apply_support` which currently isn't well designed. Have a look at issue #20404 if you have time.

Reply:
> No, I didn't. After installing librsvg I still get other errors, but I don't care as long as it worked for you. 

When i first built the documentation i had to install librsvg and imagemagick along with sphinx, as far as i remember. My terminal did tell me what dependencies i was missing. 
>Next I'd like to improve `apply_support` which currently isn't well designed. Have a look at issue #20404 if you have time.

Sure, I'll look into it!